### PR TITLE
Task 6 redux

### PR DESCRIPTION
**Pull request by katefaith**

### CODE REVIEW

File: src/components/Forms/Form.jsx [hunk 1]
```diff
@@ -1,85 +1,119 @@
...
+          <input className="form__input" type="text" id="id" placeholder="Movie ID" disabled value={movie.id} />
+        </div>
+      )}
 
-    {id && (
       <div className="form__field">
-        <label className="form__label" htmlFor="id">Movie ID</label>
-        <input className="form__input" type="text" id="id" placeholder="Movie ID" disabled value={id} />
+        <label className="form__label" htmlFor="title">Title</label>
+        <input className="form__input" type="text" id="title" onChange={handleInputChange} placeholder="Movie Title" value={movie.title} />
```
Comment (mmarkelov) [inline src/components/Forms/Form.jsx:59]: You just don't need `type="text"` - this is default value

File: src/components/Forms/Form.jsx [hunk 2]
```diff
@@ -1,85 +1,119 @@
...
-      <button className="form__button  form__button--reset" type="reset">reset</button>
-      <button className="form__button  form__button--submit" type="submit">{btnText}</button>
-    </div>
-  </form>
-);
+
+      <div className="form__field">
+        <label className="form__label" htmlFor="release-date">Release date</label>
+        <input className="form__input" type="date" id="release_date" onChange={handleInputChange} placeholder="Select Date" value={movie.release_date} />
+      </div>
```
Comment (mmarkelov) [inline src/components/Forms/Form.jsx:65]: Also i think that you can create component for one field to reduce code

File: src/components/Sorting/sorting.jsx [hunk 3]
```diff
@@ -1,14 +1,33 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMovies, setSortParam } from '../../redux/movies/actions';
+import { getCurrentRequest, getCurrentFilterParam } from '../../redux/movies/selectors';
+import { sortingItems } from '../../mockedData';
 
 import './sorting.scss';
 
-import { sortingItems } from '../../mockedData';
+const dictionary = {
```
Comment (mmarkelov) [inline src/components/Sorting/sorting.jsx]: `dictionary` does not have much sense. `sortParams` will be better

File: src/redux/crud/sagas.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+import { call } from 'redux-saga/effects';
+
+async function addMovie(movie) {
+  await axios.post('http://localhost:4000/movies', movie);
```
Comment (mmarkelov) [inline src/redux/crud/sagas.js]: it's better to move path to constant:
```js
const API = 'http://localhost:4000/'
```

File: src/components/Sorting/sorting.jsx (hunk 3 above)
Comment (katefaith) [reply src/components/Sorting/sorting.jsx]: fixed

File: src/redux/crud/sagas.js (hunk 4 above)
Comment (katefaith) [reply src/redux/crud/sagas.js]: fixed

**Pull request merged**